src/test/java/tests/NegativeSmokeTest.java

```diff
@@ -0,0 +1,77 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.DominoPage;
+import pages.MainPage;
+
+public class NegativeSmokeTest extends BaseTest {
+
+    @Test (description = "Test to check system behavior using the correct password and incorrect login")
+    public void setIncorrectLoginTest() {
+        logger.info("setIncorrectLoginTest is started...");
```

Comment:
please remove logging of start & end of the test from the actual test method. Implement that inside TestListeners (consider that as a next step in developing TAF)
Please do not include TestListeners changes into the current PR, create a separate one

Reply:
Noted